Source/Scene/Expression.js

```diff
@@ -52,6 +52,21 @@ define([
         setEvaluateFunction(this);
     }
 
+    function replaceVariables(expression) {
```

Comment:
I still think we should consider a regex here.  Can you ask @tfili how hard it would be?  I suspect it is easy.

The other thing we can consider is modifying `jsep` to allow for `${` and `}` to indicate variables, just like it can already parse functions.  This could probably be done in an elegant way that lets us add new notation later.